target/linux/mvebu/base-files/lib/preinit/05_wdmc_config

```diff
@@ -0,0 +1,21 @@
+preinit_mount_config() {
```

Comment:
base-files should now be created per subtarget, so this should be target/linux/mvebu/cortexaXX/base-files/lib/preinit/05_wdmc_config